tests/ast/conftest.py

```diff
@@ -35,6 +35,7 @@ def pytest_configure(config):
     pytest.unparser_jar = config.getoption("--unparser-jar")
     if not os.path.exists(pytest.unparser_jar):
         pytest.unparser_jar = None
+        logging.error(f"Unparser JAR not found at {pytest.unparser_jar}")
```

Comment:
Error when trying to run `--update-expectations` but no jar file is found. When no jar file is found, the expected unparser output is updated to an empty string in every file.

Reply:
nit: Add `.` at end of sentence. Maybe add explanation/action user should take to fix this error.